### PR TITLE
521 implement boostrap element dropdown

### PR DESCRIPTION
**Pull request by daniilkopylov**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                .cssClass("dropdown")
+                .tag(is("div"));
+        simpleDropdown.dropdownMenu.is()
+                .core()
+                .cssClass("dropdown-menu")
+                .attr("x-placement", "bottom-start")
+                .tag(is("div"));
+        simpleDropdown.dropdownToggle.is()
+                .core()
+                .cssClass("btn btn-primary dropdown-toggle")
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java]: please divide to checks for each class name (it makes more clear errors when it happens)       
```
simpleDropdown.dropdownToggle.is()
                .core()
                .cssClass("btn")
                .cssClass("btn-primary")
                .cssClass("dropdown-toggle")
```

Comment (daniilkopylov) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java]: fixed

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java [hunk 2]
```diff
@@ -0,0 +1,257 @@
+package io.github.epam.bootstrap.tests.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.*;
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java]: wildcard

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    }
+
+    @Test
+    public void dropLeftDropdownIsValidationTest() {
+        dropLeftDropdown.is()
+                .core()
+                .cssClass(containsString("dropleft"));
+        dropLeftDropdown.dropdownMenu.is()
+                .core()
+                .css("left", is("auto"))
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java]: please create constant variables for left and right class names to not repeat it again and again as raw text

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+                .core()
+                .css("right", is("auto"))
+                .css("left", is("100%"));
+    }
+
+    @Test
+    public void dropLeftDropdownIsValidationTest() {
+        dropLeftDropdown.is()
+                .core()
+                .cssClass(containsString("dropleft"));
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java]: just hasClass()

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+    }
+
+    @Test
+    public void getValueTest() {
+        assertEquals(simpleDropdown.getValue(), DROPDOWN);
+    }
+
+    @Test
+    public void itemsTest() {
+        assertEquals(simpleDropdown.items.getValue(), ITEMS_VALUES);
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java]: need jdi assert

**Pull request merged**